src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { useContext } from 'react';
+import { StateContext } from '../../TodosContext';
+import { TodoItem } from '../TodoItem/TodoItem';
+import { Status } from '../../types/enums/Status';
+
+export const TodoList: React.FC = () => {
+  const { todos, filter } = useContext(StateContext);
+
+  const todoItems: JSX.Element[] = [];
+
+  todos.forEach((todo) => {
+    switch (filter) {
+      case Status.All:
+        todoItems.push(<TodoItem key={todo.id} todoItem={todo} />);
+        break;
+
+      case Status.Active:
+        if (!todo.completed) {
+          todoItems.push(<TodoItem key={todo.id} todoItem={todo} />);
+        }
+
+        break;
+
+      case Status.Completed:
+        if (todo.completed) {
+          todoItems.push(<TodoItem key={todo.id} todoItem={todo} />);
+        }
+
+        break;
+
+      default:
+        todoItems.push(<TodoItem key={todo.id} todoItem={todo} />);
+        break;
+    }
+  });
```

Comment:
don't quite understand what are you doing here)
Why don't you map todos directly in your jsx?